src/app/controllers/ShiftController.ts

```diff
@@ -0,0 +1,154 @@
+import { App } from "../App";
+import { MutateResult } from "../links/synchronouslinks/GraphQLLink";
+
+export enum ShiftTimePeriodType {
+    WORK = "WORK",
+    BREAK = "BREAK",
+}
```

Comment:
@bholagabbar This was why I wanted to commit the generated enums.ts file, without committing it you need to duplicate it anyways to use in type definitions which defeats the point.